bundle/DependencyInjection/NetgenEzPlatformSearchExtraExtension.php

```diff
@@ -34,5 +44,20 @@ public function load(array $configs, ContainerBuilder $container)
 
         $loader->load('search/common.yml');
         $loader->load('persistence.yml');
+
+        $this->processExtensionConfiguration($configs, $container);
+    }
+
+    private function processExtensionConfiguration(array $configs, ContainerBuilder $container)
+    {
+        $configuration = $this->getConfiguration($configs, $container);
+
+        if ($configuration === null) {
+            return;
+        }
+
+        $configuration = $this->processConfiguration($configuration, $configs);
+
+        $container->setParameter('netgen_ez_platform_search_extra_configuration', $configuration);
```

Comment:
Instead of using a single param for the entire config, it would be better and closer to Symfony standards to use discrete parameters for specific config entries. e.g. `netgen_ez_platform_search_extra.indexable_field_type.ezxmltext.override`, `netgen_ez_platform_search_extra.indexable_field_type.ezxmltext.override.short_text_limit` and so on.